### PR TITLE
chore: remove Cargo.lock from docker ignore

### PR DESCRIPTION
**Pull request by lauti7**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .vscode
 .gitignore
 target
-Cargo.lock
 Dockerfile
+.gitignore
```
Comment (guidota) [inline .dockerignore:5]:
```suggestion

```

File: .dockerignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .vscode
 .gitignore
 target
-Cargo.lock
 Dockerfile
+README.md
+.git
```
Comment (jmoguilevsky) [inline .dockerignore:6]:
```suggestion
.git

```

Comment (jmoguilevsky) [inline .dockerignore:6]: asd
```suggestion
.git

```

Comment (jmoguilevsky) [inline .dockerignore:6]:
```suggestion
.git

```

**Review by jmoguilevsky — APPROVED** [review]

**Pull request merged**